pycuda/reduction.py

```diff
@@ -228,6 +228,7 @@ def __call__(self, *args, **kwargs):
             s2_func = kernel_wrapper(s2_func)
 
         stream = kwargs.get("stream")
+        out = kwargs.pop("out", None)
```

Comment:
Could you add this to the docs?

Reply:
Done - there was no __call__ doc so I added it and expanded the example